JsonBuilder/JsonBuilder/JsonBuilder.cs

```diff
@@ -0,0 +1,225 @@
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="jsonString">JSON</param>
+        /// <returns>Ture if is valid, otherwise fase.</returns>
+        public static bool IsValidJson(string jsonString)
+        {
+            jsonString = jsonString.Trim();
+            if (jsonString.StartsWith("{") && jsonString.EndsWith("}") || //For object
+                jsonString.StartsWith("[") && jsonString.EndsWith("]")) //For array
+                try
+                {
+                    JavaScriptSerializer.DeserializeObject(jsonString);
+                    return true;
+                }
+                catch
+                {
+                    //Debug.WriteLine(jex.Message);
+                    return false;
+                }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Convert JSON string to .NET object
+        /// </summary>
+        /// <param name="jsonString">JSON string</param>
+        /// <returns>Dictionary Representing JSON Properties</returns>
+        private static dynamic ConvertToObject(string jsonString)
+        {
+            if (!IsValidJson(jsonString))
+                throw new Exception("Not Valid Json Object!");
+
+            var jsDes = JavaScriptSerializer.DeserializeObject(jsonString);
+            return jsDes;
+        }
+
+
+        /// <summary>
+        /// Convert JSON string to .NET object
+        /// </summary>
+        /// <param name="jsonString">JSON string</param>
+        /// <returns>The property value your require. example: ToObject(Person).HisCourses</returns>
+        public static Dictionary<string, dynamic> ToObject(string jsonString)
+        {
+            if (!IsValidJson(jsonString))
+                throw new Exception("Not Valid Json Object!");
+
+            var jsDes = JavaScriptSerializer.Deserialize<dynamic>(jsonString);
+            return jsDes;
+            #region MyRegion
+
+            //var jsDes = JavaScriptSerializer.DeserializeObject(jsonString) as IDictionary<string, dynamic> ?? throw new Exception("");
+            //var objToReturn = new ExpandoObject() as IDictionary<string, dynamic>;
+            //foreach (var jsDe in jsDes)
+            //    objToReturn[jsDe.Key] = jsDe.Value;
+
+            #endregion
+        }
+
+        /// <summary>
+        /// Check the JSON object match the Generic class you passed or not.
+        /// </summary>
+        /// <typeparam name="T">Class</typeparam>
+        /// <returns>True if the JSON object match the class, false otherwise</returns>
+        public bool IsMatchThisClass<T>() where T : class
+        {
+            var properties = typeof(T).GetProperties();
+            var expandoDict = this._currentObject as IDictionary<string, dynamic>;
+            return expandoDict.Keys.Count == properties.Length && properties.All(propertyInfo => expandoDict.Keys.Contains(propertyInfo.Name));
+        }
+
+        /// <summary>
```

Comment:
Remove the comments for all `private` methods

Reply:
Done